.circleci/config.yml

```diff
@@ -0,0 +1,127 @@
+# iOS CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/ios-migrating-from-1-2/ for more details
+#
+version: 2
+jobs:
+  build:
+
+    # Specify the Xcode version to use
+    macos:
+      xcode: "9.3.0"
+    shell: /bin/bash --login -eo pipefail
+    environment:
+      - PROJECT_NAME: "ZappAppConnector"
+      - SCHEME_NAME: "ZappAppConnector"
+      - SCRIPTS_LOCAL_FOLDER: "~/.scripts-iOS"
+      - SCRIPTS_REPO_PATH: "git@github.com:applicaster/Scripts-iOS.git"
+      - DEPLOY_VERSION: "0.0.24"
+      - DEPLOY_COMMAND: "~/.scripts-iOS/circleCi/deploySingleFramework.sh"
+      #fastlane variables
+      - LC_ALL: "en_US.UTF-8"
+      - LANG: "en_US.UTF-8"
+    working_directory: ~/ZappAppConnector-iOS
+    steps:
+      - run:
+          name: Ruby - set version
+          command:  echo "ruby-2.4" > ~/.ruby-version
```

Comment:
100% sure about that? 2.4 is not too cutting edge?

Reply:
this is the default version installed on ci macos machines - using anything else would seriously increase build time